src/sql/workbench/services/connection/browser/connectionController.ts

```diff
@@ -47,24 +47,24 @@ export class ConnectionController implements IConnectionComponentController {
 			onCreateNewServerGroup: () => this.onCreateNewServerGroup(),
 			onAdvancedProperties: () => this.handleOnAdvancedProperties(),
 			onSetAzureTimeOut: () => this.handleonSetAzureTimeOut(),
-			onFetchDatabases: (serverName: string, authenticationType: string, userName?: string, password?: string, azureAccount?: string) => this.onFetchDatabases(
-				serverName, authenticationType, userName, password, azureAccount).then(result => {
+			onFetchDatabases: (serverName: string, authenticationType: string, userName?: string, password?: string, token?: string) => this.onFetchDatabases(
+				serverName, authenticationType, userName, password, token).then(result => {
 					return result;
 				}),
 			onAzureTenantSelection: (azureTenantId?: string) => this.onAzureTenantSelection(azureTenantId),
 		}, providerName);
 		this._providerName = providerName;
 	}
 
-	protected async onFetchDatabases(serverName: string, authenticationType: string, userName?: string, password?: string, azureAccount?: string): Promise<string[]> {
+	protected async onFetchDatabases(serverName: string, authenticationType: string, userName?: string, password?: string, token?: string): Promise<string[]> {
```

Comment:
Since you're making this stuff generic it's less obvious what the token is supposed to be - so I'd strongly suggest adding some function doc comments here and elsewhere you've made similar changes that explains what the "token" is